openclsim/core/identifiable.py

```diff
@@ -21,6 +21,6 @@ class Identifiable:
     """
 
     def __init__(self, name: str, ID: str = None, *args, **kwargs):
-        # super().__init__(*args, **kwargs)  # <= Unnecessary
```

Comment:
Definitely not Unnecessary. Maybe it is not used in your implementation but please don't remove it. If removed this will break inhertance.

If you dont use it, its also not hurting